lua/nvim-treesitter/compat.lua

```diff
@@ -15,4 +24,28 @@ function M.parse_query(lang, query)
   return (tsq.parse or tsq.parse_query)(lang, query)
 end
 
+function M.add_predicate(name, handler, force)
+  return (tsq.add_predicate or ts.add_predicate)(name, handler, force)
```

Comment:
You shouldn't need these. Just adapt the code to `vim.treesitter.query.add_predicate` which existed before.

Reply:
Ah, I assumed since the long message said "move" that they haven't.

Reply:
`(were they even used...)`

Reply:
What do you mean? They have always existed in the query module.

Reply:
I meant that I added the shims, but it isn't used in this repo anyways, and they have always existed as you said, making them even more pointless.